src/components/AttachmentModal.js

```diff
@@ -79,7 +79,10 @@ class AttachmentModal extends PureComponent {
             return;
         }
 
-        this.props.onConfirm(this.state.file);
+        if (this.props.onConfirm) {
+            this.props.onConfirm(this.state.file);
```

Comment:
This change is unrelated. I just noticed that if we press Enter on a regular attachment modal it throws a `TypeError` so I fixed that up.